src/components/utils/rhf-inputs/table-inputs/table-numerical-input.js

```diff
@@ -94,17 +96,24 @@ export const TableNumericalInput = ({
             }}
             InputProps={{
                 endAdornment: (
-                    <InputAdornment position="end">
-                        {/** we add the clear button only if the previous value is different from the current value **/}
-                        <IconButton
-                            onClick={handleClearValue}
-                            style={{
-                                visibility: clearable ? 'visible' : 'hidden',
-                            }}
-                        >
-                            <ClearIcon />
-                        </IconButton>
-                    </InputAdornment>
+                    <>
```

Comment:
Why do you add this level ?

Reply:
It's actually unnecessary, i've removed it